ansible_ai_connect/ai/api/views.py

```diff
@@ -450,6 +455,31 @@ def write_to_segment(
                     user,
                     ansible_extension_version,
                 )
+        if role_generation_action_data:
+            action = int(role_generation_action_data.get("action"))
+            from_page = role_generation_action_data.get("fromPage", 0)
+            to_page = role_generation_action_data.get("toPage", 0)
+            wizard_id = str(role_generation_action_data.get("wizardId", ""))
+            event = {
+                "action": action,
+                "wizardId": wizard_id,
+                "fromPage": from_page,
+                "toPage": to_page,
+                "modelName": model_name,
+            }
+            send_segment_event(event, "roleGenerationAction", user)
+            if False and from_page > 1 and action in [1, 3]:
```

Comment:
Is this deliberate? You're not sending Segment Events for page transitions.

Reply:
Yes, Aparna needs to approve the new event first.